tasks/javascript/easy/sum.js

```diff
@@ -1,4 +1,7 @@
 // JavaScript - Easy
 
 // TODO: Write a function that returns the sum of two numbers
+var a = 1;
+var b =2;
+console.log(a+b);
```

Comment:
The issue was about creating a function named `sum` that returns the sum of two numbers, but there is not any function here. Also using `var` is not recommended, you should always use `let` or `const`. 

Correct solution would have been something like this:
```bash
const sum = (a, b) => a + b;
const num1 = 1;
const num2 = 2;
const result = sum(num1, num2);
console.log(`The sum of ${num1} and ${num2} is ${result}.`);
```